my-contacts/src/Contact.js

```diff
@@ -0,0 +1,71 @@
+    const emailIconD = "M26.6623488,24.8877168 L18.6851446,19.0567597 L17.3035985,20.0988213 C16.595261,20.6331004 15.6195993,20.6368481 14.9071782,20.1080263 L13.4035686,18.9919146 L5.33765124,24.8877168 C5.54494915,24.9604486 5.76785571,25 6,25 L26,25 C26.2321443,25 26.4550509,24.9604486 26.6623488,24.8877168 Z M27.9852647,23.2440071 C27.9949926,23.1640323 28,23.0826009 28,23 L28,12.030825 L20.4429249,17.7309153 L27.9852647,23.2440071 Z M4.01473531,23.2440071 L11.63149,17.6765215 L4,12.0117563 L4,23 C4,23.0826009 4.00500744,23.1640323 4.01473531,23.2440071 Z M27.9382115,9.50476558 C27.717852,8.63973495 26.933635,8 26,8 L6,8 C5.06778797,8 4.28453891,8.63778632 4.06279998,9.50081181 L16.0987395,18.434948 L27.9382115,9.50476558 Z M6,6 L26,6 C28.209139,6 30,7.790861 30,10 L30,23 C30,25.209139 28.209139,27 26,27 L6,27 C3.790861,27 2,25.209139 2,23 L2,10 C2,7.790861 3.790861,6 6,6 Z";
+
+    let fullName = props.firstName + " " + props.lastName;
+
+    let phoneHref = "tel:" + props.phone;
+    let emailHref = "mailto:" + props.email;
+
+    return (
+        <li className={styles.contact}>
+            <img alt="profile picture" className={styles.avatar}
+                 src={props.image}/>
+
+            <div className={styles.contactDetails}>
+                <strong className={styles.name}>{fullName}</strong>
+                <p className={styles.company}>{props.company}</p>
+
+                <div className={styles.mobileAnchorContainer}>
```

Comment:
Looks like here youre rendering both mobile and desktop code.  this is fine on this scale,  but when and if things scale up you may need to conditionally render them. It would be interesting to figure out how to do that in a functional react component.

Reply:
I agree. We did briefly discuss some possible solutions. One I could see could be tracking mobile/web view as a state and rendering the specific html needed based on the state. There's probably also a way to do it exclusively with css without hiding half of the html. Would have to think some more about it though